File: app/src/main/java/com/bank/services/ui/statements/domain/model/Statements.java
package com.bank.services.ui.statements.domain.model;

import java.util.Date;
import java.util.List;

public class Statements {

    public String title;
    public String desc;
    public Date date;
    public Double value;


    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Double getValue() {
        return value;
    }

    public void setValue(Double value) {
        this.value = value;
    }



    //java.text.DateFormat dateFormat = android.text.format.DateFormat
    // .getDateFormat(getApplicationContext());

}

File: SantanderTest/app/src/main/java/com/androiddeveloper/santanderTest/data/model/statements/Statements.kt
package com.androiddeveloper.santanderTest.data.model.statements

data class Statements(var statementList: List<Item>)

data class Item(
    var title: String,
    var desc: String,
    var date: String,
    var value: Double
)

data class ItemDTO(
    var title: String = "",
    var desc: String = "",
    var date: String = "",
    var value: String = ""
)